dev-util/hyprwayland-scanner/hyprwayland-scanner-0.2.0.ebuild

```
@@ -0,0 +1,25 @@
+# Copyright 1999-2024 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+DESCRIPTION="A Hyprland implementation of wayland-scanner, in and for C++"
+
+HOMEPAGE="https://github.com/hyprwm/hyprwayland-scanner/"
+
+inherit cmake
+if [[ ${PV} == 9999 ]] ; then
+	EGIT_REPO_URI="https://github.com/hyprwm/hyprwayland-scanner.git"
+	inherit git-r3
+else
+	SRC_URI="https://github.com/hyprwm/hyprwayland-scanner/archive/refs/tags/v${PV}.tar.gz -> ${P}.tar.gz"
```

Simplify by removing `refs/tags/` and add `.gh` to distfile name
```suggestion
	SRC_URI="https://github.com/hyprwm/hyprwayland-scanner/archive/v${PV}.tar.gz -> ${P}.gh.tar.gz"
```